src/ethereum/ethNetwork.js

```diff
@@ -41,7 +41,6 @@ import {
   type FetchGetAmberdataApiResponse,
   type RpcResultString,
   asAlethioAccountsTokenTransfer,
-  asAmberdataAccountsFuncs,
```

Comment:
Duh I didn't catch these wouldn't be used anymore. Can you remove them the ethTypes too?